src/Compilers/VisualBasic/Portable/Lowering/StateMachineRewriter/StateMachineStates.vb

```diff
@@ -9,10 +9,19 @@ Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
 
 Namespace Microsoft.CodeAnalysis.VisualBasic
 
-    Friend Module StateMachineStates
-        Public FinishedStateMachine As Integer = -2
-        Public NotStartedStateMachine As Integer = -1
-        Public FirstUnusedState As Integer = 0
-    End Module
+    Friend NotInheritable Class StateMachineStates
```

Comment:
>Friend NotInheritable Class StateMachineStates [](http://example.com/codeflow?start=4&length=46)

Could this be an enum?

Reply:
It could be. I plan to share this type between C# and VB in a follow-up PR and can make it an enum then.

https://github.com/dotnet/roslyn/pull/61356/files#diff-7e5e50fcae8e80c1bb1432f401b5e65e6eeb365cd5439adb10bcb7cb1246e52a